src/panels/config/script/ha-script-editor.js

```diff
@@ -254,7 +259,7 @@ class HaScriptEditor extends LocalizeMixin(NavigateMixin(PolymerElement)) {
     if (
       this.dirty &&
       // eslint-disable-next-line
-      !confirm("You have unsaved changes. Are you sure you want to leave?")
+      !confirm("[[localize('ui.panel.config.script.editor.unsaved_confirm')]]")
```

Comment:
This one is likely more general than script editor

Reply:
Should I put it below `ui.common` then? I have oriented myself on the automation editor's translation. In addition, basically every other section has it's own strings, too (just search for `delete_confirm`or `confirm_delete` in `en.json`).

Reply:
Interesting...we should clean that up. We're just making more work for those who need to translate. I think maybe ui.panel.config.editor.common?

Reply:
Alright. Will use `ui.panel.config.common.editor`, though, to keep up with the scheme `ui.panel.config.SECTION.editor`.